wolfcrypt/test/test.c

```diff
@@ -3446,7 +3446,7 @@ int hmac_blake2b_test(void)
             continue; /* cavium can't handle short keys, fips not allowed */
 #endif
 
-    #if defined(HAVE_CAVIUM) && !defined(HAVE_CAVIUM_V)
+    #if !defined(HAVE_CAVIUM_V)
```

Comment:
Can we change this to not run the entire test for the Blake2 case with Nitrox III instead of just not doing the init?

Reply:
I think the test should remain unless we completely remove support for Blake2 with HMAC in the case that HAVE_CAVIUM && !HAVE_CAVIUM_V.